examples/multiturn_demo/tools/fake_tool_worker.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2025 The RLinf Authors.
```

Comment:
Why aren't workers in `rlinf/workers`? Is it because these workers are not general-purpose to be put there?